app/templates/my-sessions/view.hbs

```diff
@@ -7,12 +7,14 @@
       {{/if}}
     {{/link-to}}
     {{#each model.speakers as |speaker|}}
-      {{#link-to 'events.view.speakers.edit' model.event.id speaker.id}}
-        <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Edit Speaker- '}}{{speaker.name}}</button>
-        {{#if device.isMobile}}
-          <div class="ui hidden fitted divider"></div>
-        {{/if}}
-      {{/link-to}}
+      {{#if (and (eq speaker.email authManager.currentUser.email) (not-eq authManager.currentUser.isAnAdmin true))}}
```

Comment:
Admin doesn't have access to my-sessions route for any given user, so remove the admin part.

Reply:
@shubham3211 any updates?